src/platform/packages/shared/response-ops/rule_form/src/rule_definition/rule_definition.test.tsx

```diff
@@ -112,6 +112,8 @@ const { useRuleFormState, useRuleFormDispatch } = jest.requireMock('../hooks');
 
 const mockOnChange = jest.fn();
 
+const render = (toRender: ReactNode) => rtlRender(toRender, { wrapper: IntlProvider });
```

Comment:
What was the story behind this addition?

Reply:
Alternative implementation would be to put these copy fields in props and inject the i18n somewhere else but that seemed more complex.